@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -1512,7 +1548,7 @@ export class TileGeometryCreator {
         const material = new MapMeshBasicMaterial({
             color: colorHex,
             visible: isVisible,
-            depthWrite: false
+            depthWrite: true
```

Comment:
Why is that?

Reply:
opps, maybe this is a left over, I will check